### PR TITLE
Update to 0.4.4

### PR DESCRIPTION
**Pull request by lorepirri**

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -40,19 +38,14 @@ test:
...
-    - pip check
-    # Tests restricted due to modin-ray and ray-defaults unavailability.
-    # Switching off tests for features not implemented yet (NotImplementedError)
-    - pytest -vvv --capture=tee-sys tests -k "not (test_union or test_window_sum_partition_by or test_order_by_limit or test_with)"  # [py<311 and (linux64 or win)]
   requires:
     - pip
+    - cloudpickle >=1.4.0
     - dask
     # modin-ray and ray-default: no support for python 3.11 and missing on all archs except for linux-64 and win-64
     - modin-ray >=0.8.1.1  # [py<311 and (linux64 or win)]
```
Comment (JeanChristopheMorinPerso) [inline recipe/meta.yaml]: If you don't run the tests, all of these are unnecessary.
```suggestion
```

File: recipe/meta.yaml [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: e0ed05b88e321ea9935874377bda11339c90f1469f34344e9b41d16b8088e136
 
 build:
-  # s390x is missing triad  
-  skip: true  # [py<37 or s390x]
   number: 0
-  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation --ignore-installed -vvv
+  # s390x is missing triad  
+  skip: true  # [py<38 or s390x]
```
Comment (JeanChristopheMorinPerso) [inline recipe/meta.yaml]:
```suggestion
  skip: true  # [py<37 or s390x]
```
https://inspector.pypi.io/project/qpd/0.4.4/packages/81/51/82f1751e3aea61edf0c36e1bb8284e57f0856b550f9bc44be42d61335bf2/qpd-0.4.4.tar.gz/qpd-0.4.4/setup.py#line.58

File: recipe/meta.yaml [hunk 3]
```diff
@@ -27,11 +27,9 @@ requirements:
     - antlr4-python3-runtime >=4.11.1,<4.12
     - pandas >=1.2.0
     - python
-    - triad >=0.8.0
+    - triad >=0.9.0
```
Comment (JeanChristopheMorinPerso) [inline recipe/meta.yaml:29]:
```suggestion
    - triad >=0.9.0
  run_constrained:
    - cloudpickle >=1.4.0
```

**Review by JeanChristopheMorinPerso — CHANGES_REQUESTED** [review]
Please use the default PR template when creating a PR.

File: recipe/meta.yaml [hunk 4]
```diff
@@ -40,19 +40,10 @@ test:
...
     - qpd_test
     - qpd_version
-  commands:
-    - pip check
-    # Tests restricted due to modin-ray and ray-defaults unavailability.
-    # Switching off tests for features not implemented yet (NotImplementedError)
-    - pytest -vvv --capture=tee-sys tests -k "not (test_union or test_window_sum_partition_by or test_order_by_limit or test_with)"  # [py<311 and (linux64 or win)]
   requires:
     - pip
-    - dask
```
Comment (JeanChristopheMorinPerso) [inline recipe/meta.yaml:50]: Umm, it looks like this is needed for importing `qpd_dask`, my bad.

**Review by JeanChristopheMorinPerso — APPROVED** [review]

File: recipe/meta.yaml [hunk 5]
```diff
@@ -40,19 +40,13 @@ test:
     - qpd_ray  # [py<311 and (linux64 or win)]
     - qpd_test
     - qpd_version
-  commands:
-    - pip check
-    # Tests restricted due to modin-ray and ray-defaults unavailability.
-    # Switching off tests for features not implemented yet (NotImplementedError)
-    - pytest -vvv --capture=tee-sys tests -k "not (test_union or test_window_sum_partition_by or test_order_by_limit or test_with)"  # [py<311 and (linux64 or win)]
```
Comment (boldorider4) [inline recipe/meta.yaml:47]: whatever happened to the tests?

**Comment by JeanChristopheMorinPerso** [comment]
Please wait before merging this. We've got a nasty conflict between qpd and omegaconf:
```
The following packages are incompatible
├─ fugue is installable with the potential options
│  ├─ fugue 0.8.3 would require
│  │  └─ python >=3.8,<3.9.0a0 , which can be installed;
│  ├─ fugue 0.8.3 would require
│  │  └─ python >=3.10,<3.11.0a0 , which can be installed;
│  ├─ fugue 0.8.3 would require
│  │  ├─ python >=3.11,<3.12.0a0 , which conflicts with any installable versions previously reported;
│  │  └─ qpd >=0.4.0  with the potential options
│  │     ├─ qpd 0.4.1 would require
│  │     │  └─ antlr4-python3-runtime >=4.11.1,<4.12  with the potential options
│  │     │     ├─ antlr4-python3-runtime 4.11.1, which can be installed;
│  │     │     ├─ antlr4-python3-runtime [4.11.1|4.9.3] would require
│  │     │     │  └─ python >=3.10,<3.11.0a0 , which can be installed;
│  │     │     ├─ antlr4-python3-runtime 4.11.1 would require
│  │     │     │  └─ python >=3.12,<3.13.0a0 , which can be installed;
│  │     │     ├─ antlr4-python3-runtime [4.11.1|4.9.3] would require
│  │     │     │  └─ python >=3.8,<3.9.0a0 , which can be installed;
│  │     │     └─ antlr4-python3-runtime [4.11.1|4.9.3] would require
│  │     │        └─ python >=3.9,<3.10.0a0 , which can be installed;
│  │     ├─ qpd 0.4.1 would require
│  │     │  └─ python >=3.10,<3.11.0a0 , which can be installed;
│  │     ├─ qpd 0.4.1 would require
│  │     │  └─ python >=3.12,<3.13.0a0 , which can be installed;
│  │     ├─ qpd 0.4.1 would require
│  │     │  └─ python >=3.8,<3.9.0a0 , which can be installed;
│  │     └─ qpd 0.4.1 would require
│  │        └─ python >=3.9,<3.10.0a0 , which can be installed;
│  └─ fugue 0.8.3 would require
│     └─ python >=3.9,<3.10.0a0 , which can be installed;
├─ omegaconf is installable with the potential options
│  ├─ omegaconf 2.3.0 would require
│  │  └─ python >=3.8,<3.9.0a0 , which can be installed;
│  ├─ omegaconf 2.3.0 would require
│  │  └─ python >=3.10,<3.11.0a0 , which can be installed;
│  ├─ omegaconf 2.3.0 would require
│  │  ├─ antlr4-python3-runtime 4.9.*  with the potential options
│  │  │  ├─ antlr4-python3-runtime 4.9.3 conflicts with any installable versions previously reported;
│  │  │  ├─ antlr4-python3-runtime [4.11.1|4.9.3], which can be installed (as previously explained);
│  │  │  ├─ antlr4-python3-runtime [4.11.1|4.9.3], which can be installed (as previously explained);
│  │  │  └─ antlr4-python3-runtime [4.11.1|4.9.3], which can be installed (as previously explained);
│  │  └─ python >=3.11,<3.12.0a0 , which conflicts with any installable versions previously reported;
│  └─ omegaconf 2.3.0 would require
│     └─ python >=3.9,<3.10.0a0 , which can be installed;
└─ python 3.11**  is not installable because it conflicts with any installable versions previously reported.
```
It can be summarized as `omegaconf` wants `antlr4-python3-runtime` 3.9 while `qpd` wants `antlr4-python3-runtime` 4.11.

Conda forge doesn't have this problem because their omegaconf package depends on `antlr-python-runtime` while their qpd package depends on `antlr4-python3-runtime`. Both are coming from the same feedstock (https://github.com/conda-forge/antlr-python-runtime-feedstock/blob/main/recipe/meta.yaml).

I am failing t understand how it works...

**Comment by JeanChristopheMorinPerso** [comment]
Alright, I understand now. THis can be merged.

File: recipe/meta.yaml [hunk 6]
```diff
@@ -1,20 +1,20 @@
 {% set name = "qpd" %}
-{% set version = "0.4.1" %}
+{% set version = "0.4.4" %}
```
Comment (skupr-anaconda) [inline recipe/meta.yaml]:
```suggestion
```
Empty line

File: recipe/meta.yaml (hunk 5 above)
Comment (skupr-anaconda) [reply recipe/meta.yaml:47]: @lorepirri  What if you add `--ignore=tests/unit/qpd_ray`, see https://inspector.pypi.io/project/qpd/0.4.4/packages/81/51/82f1751e3aea61edf0c36e1bb8284e57f0856b550f9bc44be42d61335bf2/qpd-0.4.4.tar.gz/qpd-0.4.4/setup.cfg#line.6

Comment (lorepirri) [reply recipe/meta.yaml:47]: @boldorider4 @skupr-anaconda the folder `tests` is not included in the tarball, and there are no sources for this project.

https://inspector.pypi.io/project/qpd/0.4.4/packages/81/51/82f1751e3aea61edf0c36e1bb8284e57f0856b550f9bc44be42d61335bf2/qpd-0.4.4.tar.gz/

**Review by skupr-anaconda — APPROVED** [review]

**Review by boldorider4 — APPROVED** [review]
LGTM

**Review by boldorider4 — APPROVED** [review]
LGTM, but better add `dev_url` section

**Comment by lorepirri** [comment]
> LGTM, but better add `dev_url` section

there is no URL section

**Pull request merged**